### PR TITLE
PR to add files necessary for GitHub actions testing and test profile

### PR DESCRIPTION
**Pull request by mmeier93**
## This PR does the following:

- It adds files necessary for performing CI testing using GitHub actions (`ci.yml`, `test.config`)
- It adds a test profile in `nextflow.config`

## CI test results

Latest test was successful: https://github.com/lifebit-ai/prs/actions/runs/309113360

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install Nextflow
+        run: |
+          export NXF_VER=${{ matrix.nxf_ver }}
+          wget -qO- get.nextflow.io | bash
+          sudo mv nextflow /usr/local/bin/
+      - name: SAIGE base and gel-gwas PLINK target
+        run: |
+          nextflow run ${GITHUB_WORKSPACE} -profile test
```
Comment (cgpu) [inline .github/workflows/ci.yml:20]: Might be worth naming it

```Suggestion
          nextflow run ${GITHUB_WORKSPACE} -profile test_saige_base_gel_gwas_plink_target
```

File: nextflow.config [hunk 2]
```diff
@@ -83,4 +83,10 @@ process {
 
 }
 
+// Profiles
+
+profiles {
+  test { includeConfig 'conf/test_saige_base_gel_gwas_plink_target.config' }
```
Comment (cgpu) [inline nextflow.config:89]: To match the change above:

```Suggestion
  test_saige_base_gel_gwas_plink_target { includeConfig 'conf/test_saige_base_gel_gwas_plink_target.config' }
```

**Review by cgpu — CHANGES_REQUESTED** [review]
Looks good to me, maybe it's worth renaming with the explicit test name you created to match across all diff places.

**Comment by mmeier93** [comment]
> Looks good to me, maybe it's worth renaming with the explicit test name you created to match across all diff places.

Thanks! 👍  I made the changes you suggested and the lasts 2 tests worked. There was however an error on the third before last which seemed to be related to the GitHub API failing? (https://github.com/lifebit-ai/prs/runs/1260958654?check_suite_focus=true).

**Review by mmeier93 — COMMENTED** [review]
Thanks! 👍

**Review by cgpu — APPROVED** [review]

**Pull request merged**